### ISSUE TITLE
Can we get a boost gauge?

### ISSUE THREAD

**Issue by Dudejo** [Mogli12/GearboxAddon#427]
I'm interested to see where the new addition will go.

Would it be possible to have the boost tracked inside the HUD?

**Comment by Mogli12** [comment]
Yes, the way I increased the torque value used for fuel consumption in case of boost less than 100% was not ok.

I removed it and found another problem. Now I changed many parameters. I reduced moment of inertia to have a lower fuel consumption at constant speed. Afterwards I increased the motor brake force to have enough brake force when going downhill. And now all vehicles use the boost calculation because it is a nice way to smooth and limit motor torque if the vehicle is revving up too fast.

I hope the fuel consumption is more realistic now. And I hope that the vehicles do not accelerate too fast.

**Comment by Dudejo** [comment]
Feels pretty good when I drive around in my truck.

Although, for the purposes of the Lizard Rodeo, what flags do I set in the individual vehicle to reduce the boost delays?

**Comment by Mogli12** [comment]
The attribute is fullBoostMs, e.g. 
```xml
<gearboxMogli fullBoostMs="1500"`...>
```
or 
```xml
<transmissions>
  <transmission fullBoostMs="150" ...>
...
```
I took 150 ms for the naturally aspirated engines. This has the advantage that there is a little smoothing of torque if the vehicle cruises a speed limit.

**Comment by Dudejo** [comment]
nice =b

The only other tag I could ask for is for setting the time until no boost. The default of 1500 ms is a bit too quick, in my opinion.

https://www.youtube.com/watch?v=wsYFwv1okxs

While this video features a much lighter sports car's turbo, the car provides almost immediately maximum boost even after a gear shift. This leads me to believe that turbos retain some sort of momentum that allows maximum boost relatively quickly when accelerating.

**Comment by Jurkkis** [comment]
Boost makes even IRL engine tuning complicated, so much variables. Google for boost treshhold vs turbo lag, all those engine characteristics would be needed to model to the game for each engine, which I think is too much. Also diesel engines normally dont have butterfly valve for throttle, they breath air always as much they can and power is adjusted with amount of fuel injected. Whereas turbo petrol engines sometimes have bypass/or dumpvalves to help turbo keep momentum when throttle closes during normal (full acceleration) upshifts.

**Comment by Dudejo** [comment]
@Jurkkis 

The mod just implemented a turbo lag that works independently from the torque curve.

**Comment by Mogli12** [comment]
Hi,

This new feature is just the delay between no torque and full torque at idle RPM. The time decreases to 0 at rated RPM.

Additionally, you can control how fast or slow the max available torque limit decreases.

Regards, Stefan
